app/src/main/java/com/tolkiana/tacoapp/ProductDetailAdapter.kt

```diff
@@ -0,0 +1,68 @@
+package com.tolkiana.tacoapp
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+
+/**
+ * Created by tolkiana on 12/23/17.
+ */
+class ProductDetailAdapter(private val product: Product) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
```

Comment:
This class is a great example of using a RecyclerView adapter for rendering different things for different types of data. Really nicely done!

Reply:
Yey! Thanks again for taking the time to review!